pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -127,8 +127,9 @@ Activity:
     guid:
       type: string
       description: unique identifier for this instance of the activity
-    status:
-      type: string
+    statusCode:
```

Comment:
@fat23cat 
Pavel: This is change to user activities API/Route . 
If front-end is using "status" field, we need a front-end change to be compatible with this backend change. 
This is not summary, individual activity instance details.
"/user/%s/studies/%s/activities/%s" call that gets user activity details.

Reply:
Frontend doesn't use that field.